persistent/test/main.hs

```diff
@@ -54,32 +54,36 @@ main = hspec $ do
                 `shouldBe`
                     ( [NEL.toList helloWorldTokens, NEL.toList foobarbazTokens], mempty )
         it "works4" $ do
-            let foobarbarz = ["foo", "Bar", "baz"]
```

Comment:
These were also causing some type ambiguity errors, where with `OverloadedLists` we want to use this value here both as a `[Text]` and `NonEmpty Text`, so I have opted to re-writing the test with literals rather than trying to re-use the values. I've tried to include some more realworld looking values too while changing things here.